Source/context/context.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <glad/glad.h>
+
+#include "../input/callback.h"
+#include "../window/window.h"
```

Comment:
Unnecessary includes; you don't need anything here, you can forward declare `GLFWwindow*` and improve compilation.